meteorite_fall.js

```diff
@@ -49,6 +54,25 @@ function initializeMeteorites() {
     }
 }
 
+function checkGameOver(meteoriteX, meteoriteY, playerX, playerY, meteoriteWidth, meteoriteHeight, playerWidth) {
+    if (meteoriteY + meteoriteHeight == playerY) {
+        if (meteoriteX <= playerX + playerWidth && meteoriteX + meteoriteWidth >= playerX) {
+            isGameOver = true;
```

Comment:
In my other comment I assumed this function would return a bool saying if the game is over or not. I would make this change because:
* Currently this method violates the SRP (It does more than one thing). Because it checks if the meteor and player collide, and if so it does some stuff. So I would just refactor this method so that it only does the check. Then in the main loop I would add to the if statement.
if checkGameOver(player, meteor) {
    // do stuff
    break;
}
* I think the technical term for this is a *side affect*. But basically instead of returning something, you are changing the external var isGameOver. This should be generally avoided because it creates confusing control flow. And makes unit testing harder. So I would just return the bool value.